FLEDGE.md

```diff
@@ -746,6 +747,15 @@ The output of `generateBid()` contains the following fields:
 *   allowComponentAuction: If this buyer is taking part of a component auction, this value must be present and true, or the bid is ignored. This value is ignored (and may be absent) if the buyer is part of a top-level auction.
 * modelingSignals: A 0-4095 integer (12-bits) passed to `reportWin()`, with noising, as described in the [noising and bucketing scheme](#521-noised-and-bucketed-signals). Invalid values, such as negative, infinite, and NaN values, will be ignored and not passed. Only the lowest 12 bits will be passed.
 
+In case returning multiple bids is supported by the implementation in use,
+`generateBid` may also return up to `browserSignals.multiBidLimit` valid bid
+objects of the format above in an array.
+
+If none of the returned bids pass the k-anonymity checks, `generateBid` will be
+re-run with the input `interestGroup` filtered to contain only k-anonymous ads
+and component ads. Such re-runs are limited to returning only a single bid,
```

Comment:
I think we need to specify that it will be rerun with a multi-bid limit of one.  The explainer really has the how-to-use-this-API-in-a-semi-readable-format, and I think the fact that even when the SSP and DSP agree to always use a multi-bid-limit of X, that the bidder script still needs to support a bid limit of 1 is entirely non-obvious here.

Reply:
>... that the bidder script still needs to support a bid limit of 1 is entirely non-obvious here.

Multi-bid is not introducing a "this-is-the-kanon-second-pass" signal, right? So how would the bidder know to cap the bids in that pass?

Reply:
Got it. I missed that on the second pass `browserSignals.multiBidLimit==1`.